api-authorization/server/server.js

```diff
@@ -84,7 +84,7 @@ app.get('/api/entries', authMiddleware, async (req, res, next) => {
   }
 });
 
-app.post('/api/entries', authMiddleware, async (req, res, next) => {
+app.post('/api/entries', async (req, res, next) => {
```

Comment:
Oops, you removed `authMiddleware` from the wrong endpoint. Put `authMiddleware` on all endpoints where you want the user to be logged in before they use it. Here, you want them to be logged in before they see any entries because you want to show them only _their_ entries.

When they sign-in or sign-up they aren't yet logged in, so you do _not_ want `authMiddleware` on those endpoints.